expression/builtin_json_vec.go

```diff
@@ -581,4 +581,4 @@ func (b *builtinJSONUnquoteSig) vecEvalString(input *chunk.Chunk, result *chunk.
 		result.AppendString(res)
 	}
 	return nil
-}
+}
```

Comment:
change the code by accident?

Reply:
I think it's my master branch‘s problem.